src/m365/teams/commands/channel/channel-get.ts

```diff
@@ -44,21 +44,24 @@ class TeamsChannelGetCommand extends GraphCommand {
     }
 
     const requestOptions: any = {
-      url: `${this.resource}/beta/groups?$filter=resourceProvisioningOptions/Any(x:x eq 'Team') and displayName eq '${encodeURIComponent(args.options.teamName as string)}'`,
+      url: `${this.resource}/v1.0/groups?$filter=displayName eq '${encodeURIComponent(args.options.teamName as string)}'`,
       headers: {
         accept: 'application/json;odata.metadata=none'
       },
       responseType: 'json'
     };
 
     return request
-      .get<{ value: [{ id: string }] }>(requestOptions)
+      .get<{ value: [{ id: string, resourceProvisioningOptions: string[] }] }>(requestOptions)
       .then(response => {
-        const groupItem: { id: string } | undefined = response.value[0];
+        const groupItem: { id: string, resourceProvisioningOptions: string[] } | undefined = response.value[0];
 
         if (!groupItem) {
           return Promise.reject(`The specified team does not exist in the Microsoft Teams`);
         }
+        else if (groupItem && groupItem.resourceProvisioningOptions.indexOf("Team") === -1) {
```

Comment:
No need to use `else if (groupItem` because at this point we already verified that `groupItem` is defined, so `if (groupItem.resourceProvisioning...` would suffice